packages/cli/source/services/source-providers/file.test.ts

```diff
@@ -32,7 +32,7 @@ describe<{
 	// TODO: fix stub
 	it.skip("#install - should successfully install the plugin", async ({ source, dataPath, temporaryPath }) => {
```

Comment:
Can we now enable all the skipped tests due to stubs again?

Reply:
Separet PR.